app/views/admin/call_for_papers/show.html.haml

```diff
@@ -19,8 +19,12 @@
         %dt
           Days Left:
         %dd
-          - days = (@call_for_paper.end_date - Date.today).to_i
+          - if @conference.cfp_open?
+            - days = @call_for_paper.remaining_days
+          - else
+            - days = 0
```

Comment:
I'd move the zero check to the helper, so you don't need to check it in the view!

Sth like 

```
def remaining_days(date = Date.today)
  result = (self.end_date - date).to_i
  result > 0 ? result : 0
end
```